session/bootstrap.go

```diff
@@ -264,6 +264,12 @@ const (
 	CreateExprPushdownBlacklist = `CREATE TABLE IF NOT EXISTS mysql.expr_pushdown_blacklist (
 		name char(100) NOT NULL
 	);`
+
+	// CreateOptRuleBlacklist stores the list of disabled optimizing operations.
+	CreateOptRuleBlacklist = `CREATE TABLE IF NOT EXISTS mysql.opt_rule_blacklist (
+		name char(100) NOT NULL,
+		type char(20) check(type="logical_rule") NOT NULL
```

Comment:
Do we still need the `type` field if we separate the blacklist of rules and pushdown functions?

Reply:
Yes, in the future, after the cascade planner complete, this field can also be used for it.

Reply:
Do you mean to use this field to indicates which planner this rule belonging to?

Reply:
Yes.

Reply:
I think we don't need this field even when we want to disable some rules in the cascades planner. The rule name in cascades planner is different from the current planner. BTW, we are going to replace the current planner with the cascades planner in the near future. At that time, the only valid type is `cascades planner`. How about only use a `name` field?

Reply:
OK